cmd/nginx-ingress/flags.go

```diff
@@ -40,6 +40,11 @@ var (
 
 	watchNamespaces []string
 
+	watchSecretNamespace = flag.String("watch-secret-namespace", "",
+		`Comma separated list of namespaces the Ingress Controller should watch for secrets. By default the Ingress Controller watches all namespaces`)
```

Comment:
Maybe something like this would be better? This makes it seem like all namespaces will be watched for secrets even if `watch-namespace` is set.
```suggestion
		`Comma separated list of namespaces the Ingress Controller should watch for secrets. If this arg is not configured, the Ingress Controller watches the same namespaces for all resources. See `watch-namespace`. `)
```